### PR TITLE
Dev zhi fix2639

### PR DESCRIPTION
**Pull request by zhili1208**
https://github.com/NuGet/Home/issues/2639
adding support for using relative path for configFile flag.

### CODE REVIEW

File: src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs [hunk 1]
```diff
@@ -167,6 +167,16 @@ public string ConfigFilePath
                 useTestingGlobalPath: false);
         }
 
+        public static ISettings LoadSpecificSettings(string root, string configFileName)
```
Comment (emgarten) [inline src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs:170]: :+1:

**Comment by emgarten** [comment]
What happens if the config file doesn't exist? Does it show an error?

**Comment by zhili1208** [comment]
No, it will load appdata config, I think we should throw for this case, let me fix it

**Comment by emgarten** [comment]
Great fix, some XML comments for `LoadSpecificSettings` would be nice. :shipit:

**Pull request closed without merging**